model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -48,6 +48,20 @@ data class OrtConfiguration(
      */
     val severeIssueThreshold: Severity = Severity.WARNING,
 
+    /**
+     * Enable the usage of project-local package curations from the [RepositoryConfiguration]. If set to true apply
+     * package curations from a local .ort.yml file. These package curations are merged with package curations from
+     * the command line option, while the one from .ort.yml take precedence.
```

Comment:
Recommend to change to: If set to true, apply package curations from a local .ort.yml file before applying those specified via the command line e.g. curations from the.ort.yml take precedence.

Reply:
Done. But again used `i.e.` instead of e.g. which I think is the correct term to use here.